floris/utilities.py

```diff
@@ -90,6 +92,10 @@ def rotate_on_x3(self, theta, center_of_rotation=None):
         )
         self.x3prime = self.x3
 
+    @property
+    def coords(self):
```

Comment:
Could this be renamed? The `Vec3` class is often used for coordinates, but its meant to be a general purpose class for any 3-component vector. For example, this could represent the velocity components in a field.

Another option is to implement to `TODO` in the docstring.

Reply:
This one I might leave up to you, sorry, don't have a strong opinion...

Reply:
I did this one also, it could easily be renamed. I assumed it was coordinates because it's frequently used for them; it was a convenience feature instead of calling all the items in a number of spots throughout the code.